binaries/speed_benchmark_torch.cc

```diff
@@ -82,10 +82,15 @@ int main(int argc, char** argv) {
     }
   }
 
+  auto qengines = at::globalContext().supportedQEngines();
+  if (std::find(qengines.begin(), qengines.end(), at::QEngine::QNNPACK) != qengines.end()) {
+    at::globalContext().setQEngine(at::QEngine::QNNPACK);
+  }
   torch::autograd::AutoGradMode guard(false);
   auto module = torch::jit::load(FLAGS_model);
 
   at::AutoNonVariableTypeMode non_var_type_mode(true);
+  module.eval();
```

Comment:
Does this need to be called after NonVariableTypeMode guard?

Reply:
I think it needs to be called before calling forward. Currently my model only works when `at::AutoNonVariableTypeMode` guard is called before module load so I can't test the theory out :)